### DISCUSSION TITLE
Quasar CLI + inertia?

### DISCUSSION THREAD

**Discussion by mbnoimi** [quasarframework/quasar#15207] [General - Components / Directives / etc]
Is there any way to use these technologies together?

Laravel + Quasar CLI + Inertia

My dream in this stack

**Comment by mbnoimi** [comment]
to be honest I loved Vue because of Quasar

**Comment by metalsadman** [comment]
> Is there any way to use these technologies together?
> 
> Laravel + Quasar CLI + Inertia
> 
> My dream in this stack

i think @ldiebold have worked on same stack if i'm not mistaken.

**Reply by mbnoimi** [reply]
> i think @ldiebold have worked on same stack if i'm not mistaken.

I think @ldiebold uses [Laravel Orion](https://tailflow.github.io/laravel-orion-docs/) which doesn't use Inertia

**Reply by metalsadman** [reply]
he has used inertia.
